### PR TITLE
cluster: use initial_commit_ts in drainer config file instead of start script

### LINKED ISSUE #1682: --initial-commit-ts should not be recorded at run_drainer.sh
initial-commit-ts should be configured at drainer_servers.config.initial-commit-ts , But now ,drainer_servers.commit_ts do the same thing and has higher priority.

### PR DESCRIPTION
**Pull request by nexustar**
### What problem does this PR solve? 

- close #1682 

### Check List 

Tests 

 - Unit test
 - Integration test
 - Manual test (add detailed scripts or steps below)
 - No code

Code changes

 - Has exported function/method change
 - Has exported variable/fields change
 - Has interface methods change
 - Has persistent data change

Side effects

 - Possible performance regression
 - Increased code complexity
 - Breaking backward compatibility

Related changes

 - Need to cherry-pick to the release branch
 - Need to update the documentation

Release notes:

```release-note
NONE
```

### CODE REVIEW

File: pkg/cluster/spec/drainer.go [hunk 1]
```diff
@@ -38,7 +38,7 @@ type DrainerSpec struct {
 	DeployDir       string                 `yaml:"deploy_dir,omitempty"`
 	DataDir         string                 `yaml:"data_dir,omitempty"`
 	LogDir          string                 `yaml:"log_dir,omitempty"`
-	CommitTS        int64                  `yaml:"commit_ts" default:"-1" validate:"commit_ts:editable"`
+	CommitTS        int64                  `yaml:"commit_ts,omitempty" validate:"commit_ts:editable"` // do not use it anymore, exist for compatibility
```
Comment (srstack) [inline pkg/cluster/spec/drainer.go:41]: commit_ts:editable ?

Comment (nexustar) [reply pkg/cluster/spec/drainer.go]: yes

Comment (nexustar) [reply pkg/cluster/spec/drainer.go]: > commit_ts:editable ?

yes

**Review by AstroProfundis — APPROVED** [review]

**Pull request merged**